modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -114,19 +114,27 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 <4> The cluster name that you specified in your DNS records.
 <5> Optional parameter for providing additional configuration for the machine pool parameters for the compute and control plane machines.
 <6> Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
-<7> Optional parameter for providing an existing resource pool for machine creation. If you do not specify a value, the installation program uses the root resource pool of the vSphere cluster.
-<8> The vSphere disk provisioning method.
+<7> The path to the vSphere datastore that holds virtual machine files, templates, and ISO images. 
++
+[IMPORTANT]
+====
+You can specify the path of any datastore that exists in a datastore cluster. By default, Storage vMotion is automatically enabled for a datastore cluster. Red Hat does not support Storage vMotion, so you must disable Storage vMotion to avoid data loss issues for your {product-title} cluster.
+
+If you need to specify VMs across multiple datastores, use a `datastore` object to specify a failure domain in your cluster's `install-config.yaml` configuration file. For more information, see "VMware vSphere region and zone enablement".
+====
+<8> Optional parameter for providing an existing resource pool for machine creation. If you do not specify a value, the installation program uses the root resource pool of the vSphere cluster.
```

Comment:
For any of the "Optional parameter" callouts in this PR, even tho https://www.ibm.com/docs/en/ibm-style?topic=format-procedures is procedure/step focused (I can't find an IBM SG section explicitly discussing optional callouts like this), suggest considering the "Optional:" style (which is in wide use throughout the repo, FWIW).

Reply:
Valid point. Updated instances to ensure consistency.